keyboards/trnthsn/s6xty/readme.md

```diff
@@ -0,0 +1,27 @@
+# S6xty
+
+![trnthsn/s6xty](https://i.imgur.com/YJZxxqY.png)
+
+*A short description of the keyboard/project*
```

Comment:
Could a short description of the keyboard/PCB be placed here?

Reply:
i has pushed a new commit to add a description here